lab2/src/task4/Book.java

```diff
@@ -0,0 +1,18 @@
+package lab2.task4;
+
+public class Book {
+    private String title;
+    private String author;
+    private int year;
+
+    public Book(String title, String author, int year) {
+        this.title = title;
+        this.author = author;
+        this.year = year;
+    }
+
+    public String toString() {
+        return title + ", written by " + author
+                + ", published in " + year;
```

Comment:
Try to align the content when you break a long line, especially in toString (+ should be on the same column). 
I really appreciate that you thought of this 😍